tests/unittests/algo/test_tpe.py

```diff
@@ -403,7 +403,7 @@ def test_get_loglikelis(self, tpe: TPE):
 
 
 def _trial_to_array(trial: Trial, space: Space) -> np.ndarray:
```

Comment:
The typing would be wrong.
```suggestion
def _trial_to_array(trial: Trial, space: Space) -> list[tuple]:
```